install_config/install/prerequisites.adoc

```diff
@@ -114,12 +114,27 @@ this during or after installation.
 
 [[sizing-recommendations]]
 === Sizing Recommendations
+The following apply to production environments. Test or sample environments will
+function with the minimum requirements.
 
-In general, an {product-title} cluster should have 1 CPU core and 2.5 GB of
-memory, on top of the defaults in the table above, for each 1000 pods.
-Therefore, the recommended size of a cluster of 2000 pods would be 20 CPU cores
-and 50 GB of RAM, as well as the minimum requirements for a master host of 2 CPU
-cores and 8 GB of RAM.
+*Master Hosts*
+
+In a highly-available {product-title} cluster with external etcd, a master host
+should have 1 CPU core and 2.5 GB of memory, on top of the defaults in the table
+above, for each 1000 pods. Therefore, the recommended size of master host in an
+{product-title} cluster of 2000 pods would be 20 CPU cores and 50 GB of RAM, as
+well as the minimum requirements for a master host of 2 CPU cores and 8 GB of
+RAM.
+
+*Node Hosts*
+
+The size of a node host depends on the size of the workload that will be run on
+the host. As an {product-title} cluster administrator, you will need to
+calculate the expected workload, making sure to leave some room for reserve
+resources, and use this to determine the size of the node host.
+
+In general, add 30% to the maximum expected workload. This will account for 10%
```

Comment:
@bfallonf yes that's true, the 20% is meant for picking up slack in case a node fails. It just sounds a bit strange when we're talking about individual nodes, seemingly suggesting that 20% of a node could fail somehow. That figure also makes the assumption that there are at least 5 schedulable nodes in the cluster. I think in this context, when we're trying to be brief it makes more sense to just say calculate expected maximum workload per node, add 10% for overhead, and for production deployments, make sure to allocate enough resources such that a host failure doesn't restrict your maximum capacity.